app_flutter/lib/main.dart

```diff
@@ -40,9 +40,12 @@ class _BuildDashboardPageState extends State<BuildDashboardPage> {
 
   @override
   Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
     return Scaffold(
       appBar: AppBar(
         title: Text('Flutter Build Dashboard v2'),
+        backgroundColor:
+            buildState.isTreeBuilding ? theme.primaryColor : theme.errorColor,
```

Comment:
In addition to "Succeeded" and "Failed", it's useful to know how outdated is the information.